curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/add-methods-after-inheritance.md

```diff
@@ -34,13 +34,15 @@ Now instances of `Bird` will have both `eat()` and `fly()` methods:
 
 ```js
 let duck = new Bird();
-duck.eat(); // prints "nom nom nom"
-duck.fly(); // prints "I'm flying!"
+duck.eat();
+duck.fly();
 ```
 
+`duck.eat()` would display the string `nom nom nom` in the console, and `duck.fly()` would display the string `I'm flying!`.
+
 # --instructions--
 
-Add all necessary code so the `Dog` object inherits from `Animal` and the `Dog's` `prototype` constructor is set to Dog. Then add a `bark()` method to the `Dog` object so that `beagle` can both `eat()` and `bark()`. The `bark()` method should print "Woof!" to the console.
+Add all necessary code so the `Dog` object inherits from `Animal` and the `Dog's` `prototype` constructor is set to Dog. Then add a `bark()` method to the `Dog` object so that `beagle` can both `eat()` and `bark()`. The `bark()` method should print `Woof!` to the console.
```

Comment:
This is more out of my personal interest:
- How does something like this translate in other languages:
  > ...the `Dog's` `prototype` constructor is set to Dog...

Specifically, I am wondering about the apostrophe in the backticks, and the word "Dog" at the end not.

Reply:
Maybe it should be:
```suggestion
Add all necessary code so the `Dog` object inherits from `Animal` and the `Dog`'s `prototype` constructor is set to `Dog`. Then add a `bark()` method to the `Dog` object so that `beagle` can both `eat()` and `bark()`. The `bark()` method should print `Woof!` to the console.
```